scrape/scrape_test.go

```diff
@@ -4680,28 +4523,28 @@ metric: <
 		},
 		"protobuf, with only native exponential histogram": {
 			text: []string{
-				genTestCounterProto("test_metric_1", 1),
-				genTestCounterProto("test_metric_1_count", 1),
-				genTestCounterProto("test_metric_1_sum", 1),
-				genTestCounterProto("test_metric_1_bucket", 1),
+				genTestCounterProto("test_metric_1"),
+				genTestCounterProto("test_metric_1_count"),
+				genTestCounterProto("test_metric_1_sum"),
+				genTestCounterProto("test_metric_1_bucket"),
 				genTestHistProto("test_histogram_1", false, true),
-				genTestCounterProto("test_metric_2", 1),
-				genTestCounterProto("test_metric_2_count", 1),
-				genTestCounterProto("test_metric_2_sum", 1),
-				genTestCounterProto("test_metric_2_bucket", 1),
+				genTestCounterProto("test_metric_2"),
+				genTestCounterProto("test_metric_2_count"),
+				genTestCounterProto("test_metric_2_sum"),
+				genTestCounterProto("test_metric_2_bucket"),
 				genTestHistProto("test_histogram_2", false, true),
-				genTestCounterProto("test_metric_3", 1),
-				genTestCounterProto("test_metric_3_count", 1),
-				genTestCounterProto("test_metric_3_sum", 1),
-				genTestCounterProto("test_metric_3_bucket", 1),
+				genTestCounterProto("test_metric_3"),
+				genTestCounterProto("test_metric_3_count"),
+				genTestCounterProto("test_metric_3_sum"),
+				genTestCounterProto("test_metric_3_bucket"),
 				genTestHistProto("test_histogram_3", false, true),
 			},
 			contentType:    "application/vnd.google.protobuf",
 			hasExponential: true,
 		},
 	}
 
-	checkBucketValues := func(expectedCount int, series storage.SeriesSet) {
+	checkBucketValues := func(t testing.TB, expectedCount int, series storage.SeriesSet) {
```

Comment:
Note to reviewers: `checkXYZ` functions are used inside t.Run so we need a local t.